src/server.py

```diff
@@ -1,52 +1,102 @@
-"""Module to create a simple echo server."""
+"""Module to create a simple http server."""
 # -*- coding: utf-8 -*-
 import socket
 
 
+class Response(object):
+    """Create Response Class."""
+
+    def __init__(self, code, body, headers=None):
+        """Init Response with Status code."""
+        self.protocol = "HTTP/1.1"
+        self.code = code
+        self.status = [self.protocol + " ", code + "\r\n"]
+        self.body = body
+        if headers:
+            self.headers = headers
+
+    def return_response_string(self):
+        """Return this Response Instances's response string."""
+        blank_line = "\r\n"
+
+        response_list = self.status
+
+        try:
+            for k, v in self.headers.items():
+                response_list.append(k + ": ")
+                response_list.append(v + "\r\n")
+        except AttributeError:
+            pass
+
+        response_list.append(blank_line)
+        response_list.append(self.body)
+
+        response_string = "{}" * len(response_list)
+
+        return response_string.format(*response_list)
+
+
+def response_ok():
+    """Return Status 200 response with body and headers."""
+    headers = {"Content-Type": "text/plain"}
+    body = "This is some text -- body text"
+    response = Response("200 OK", body=body, headers=headers)
```

Comment:
This is really cool, using a class to build your responses. Very clean. I like it.